ansible/django-ansible-base
# Fix authenticator order

Fixes AAP-28034:
**Changes Made:**
- Added a `get_next_authenticator_order` function to find the next order value, which is equals to `max(order) + 1`
- Added unit test

**Testing Steps:**
1. Create a new authenticator, and confirm the followings:
- If given an order, the created authenticator's order = given order value
- If no order was given,  the created authenticator's order = max(order) + 1
2. Update/ Delete the order of the current authenticators
3. Back to step 1 and confirm the authenticator order is as expected
